packages/twenty-front/src/modules/ui/input/editor/components/CustomAddBlockItem.tsx

```diff
@@ -31,11 +31,13 @@ export const CustomAddBlockItem = ({
     const [firstElement] = currentBlockContent || [];
 
     if (firstElement === undefined) {
```

Comment:
I think the 'Add block' feature is now broken? It used to open the panel with options
<img width="257" alt="Screenshot 2024-12-29 at 15 23 40" src="https://github.com/user-attachments/assets/10d40769-7b7d-4712-8655-cdde879d0f4b" />

Reply:
Resolved in recent commit.